### PR TITLE
Adds a function to safely compute log.

### LINKED ISSUE #126: Implement `logSafe`
From `ospsuite-r`, as this function is also used by the ParameterIdentification and should be also used by reporting engine?

```
# TODO:
#
# Depending on what is decided in issue
# https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1091, change
# defaults for `base` for `.log_safe`.

#' @keywords internal
#' @noRd
.log_safe <- function(x, base = 10, epsilon = ospsuiteEnv$LOG_SAFE_EPSILON) {
  x <- sapply(X = x, function(element) {
    element <- ospsuite.utils::toMissingOfType(element, type = "double")
    if (is.na(element)) {
      return(NA_real_)
    } else if (element < epsilon) {
      return(log(epsilon, base = base))
    } else {
      return(log(element, base = base))
    }
  })

  return(x)
}
```

### PR DESCRIPTION
**Pull request by PavelBal**
Increases version number to 1.5

Fixes #126

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -25,6 +25,7 @@ export(isIncluded)
 export(isOfLength)
 export(isOfType)
 export(isSameLength)
+export(log_safe)
```
Comment (Yuri05) [inline NAMESPACE:28]: why naming mit underscore here and not `logSafe `?

Comment (PavelBal) [reply NAMESPACE:28]: Good question.... Was named like this in ospsuite, but I will rename.

File: tests/testthat/test-utilities.R [hunk 2]
```diff
@@ -40,3 +40,11 @@ test_that("Checks if `toMissingOfType()` doesn't convert other objects", {
   expect_equal(toMissingOfType(TRUE, type = "character"), TRUE)
   expect_equal(toMissingOfType(1L, type = "logical"), 1L)
 })
+
+test_that("Correct behavior of logSafe", {
+  defEpsilon <- getOSPSuiteUtilsSetting("LOG_SAFE_EPSILON")
+  inputVector <- c(NA, 1, 5, 0, -1, defEpsilon)
+
+  expectedOutput <- c(NA_real_, log(1), log(5), log(defEpsilon), log(defEpsilon), log(defEpsilon))
+  expect_equal(logSafe(inputVector), expectedOutput)
```
Comment (Yuri05) [inline tests/testthat/test-utilities.R:49]: can you extend the test for another log base?

Comment (PavelBal) [reply tests/testthat/test-utilities.R:49]: also testing for log10 now.

**Comment by Yuri05** [comment]
need to add logSafe to _pkgdown.yml topics list. After that: looks good for me.

File: R/utilities.R [hunk 3]
```diff
@@ -111,3 +111,39 @@ toMissingOfType <- function(x, type) {
 
   return(x)
 }
+
+# TODO:
```
Comment (Felixmil) [inline R/utilities.R:115]: Remove the TODO to avoid future confusion ?

**Review by Felixmil — APPROVED** [review]

**Review by Felixmil — APPROVED** [review]

**Pull request merged**